test/SlowTests/MailingList/DocumentQueryIncludeAndStreamTest.cs

```diff
@@ -88,11 +90,36 @@ public void StreamDocumentQueryWithInclude()
 
                         }
                     }
-                }).InnerException;
+                });
                 Assert.Contains("Includes are not supported by this type of query", notSupportedException.Message);
             }
         }
 
+        [RavenTheory(RavenTestCategory.Querying)]
+        [RavenData(DatabaseMode = RavenDatabaseMode.Single)]
+        public void StreamDocumentQueryWithInclude2(Options options)
+        {
+            var store = GetDocumentStore(options);
+            Setup(store);
+            Indexes.WaitForIndexing(store);
+            using (var session = store.OpenSession())
+            {
+                var query = session.Query<ProcessStep, ProcessStepIndex>(); //TODO stav: doesn't throw but doesn't stream includes docs either. is supported? should throw?
```

Comment:
What about this? is includes supported in regular streaming queries? or should this test be throwing?